bedrock/firefox/templates/firefox/android/index.html

```diff
@@ -61,6 +61,11 @@ <h2 class="heading">
           <a href="{{ url('firefox.sms') }}">Text a download link to my Android phone</a>
         </p>
       </div>
+
+      {% if l10n_has_tag('ios_announce', 'firefox/new') %}
+        <p class="ios-link">{{_('Firefox is coming soon to iOS! <a href="{url}">Sign up to learn more</a>.').format(url=url('newsletter.ios'))|safe }}</p>
```

Comment:
You'll cringe at this, but I suggest a <code>br</code> before the link (here and in the other places this string appears). You could style the link as block-level but that has other layout implications and I think a plain old line break is simpler.